src/mscorlib/shared/README.md

```diff
@@ -0,0 +1,15 @@
+# .NET Core Runtime Shared Directory
+
+This directory contains the shared sources for System.Private.CoreLib. These are shared between dotnet/corert and dotnet/coreclr.
```

Comment:
you could add links to https://github.com/dotnet/coreclr/tree/master/src/mscorlib/shared and https://github.com/dotnet/corert/tree/master/src/System.Private.CoreLib/shared